src/appleseed/renderer/modeling/light/sunlight.cpp

```diff
@@ -418,6 +437,9 @@ namespace
                 + disk_radius * p[0] * basis.get_tangent_u()
                 + disk_radius * p[1] * basis.get_tangent_v();
 
+            if (target_point != Vector3d(0.0))
+                outgoing = normalize(target_point - position);
```

Comment:
this checks for dummy target_point passed from sppmphotontracer.cpp and lighttracingsamplegenerator.cpp